### PR TITLE
CSM Operator document changes for CSM 1.9.1 patch release

### PR DESCRIPTION
**Pull request by HarishH-DELL**
# Description
CSM Operator document changes for v1.4.1

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1109 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csmoperator/release/_index.md [hunk 1]
```diff
@@ -31,4 +31,40 @@ Description: >
 - [#990 - [BUG]: X_CSI_AUTH_TYPE cannot be set in CSM Operator](https://github.com/dell/csm/issues/990)
 
 ### Known Issues
-There are no known issues in this release.
\ No newline at end of file
+There are no known issues in this release.
```
Comment (cbartoszDell) [inline content/docs/deployment/csmoperator/release/_index.md:34]: One issue is listed below this statement.

File: content/docs/deployment/csminstallationwizard/src/index.html [hunk 2]
```diff
@@ -87,7 +87,7 @@
                       <select id="csm-version" class="form-select" onchange="onCSMVersionChange(); validateInput(validateForm, CONSTANTS)" required>
                         <option value="1.7.0">CSM 1.7</option>
                         <option value="1.8.0">CSM 1.8</option>
-                        <option value="1.9.0" selected>CSM 1.9</option>
+                        <option value="1.9.0">CSM 1.9</option>
```
Comment (adarsh-dell) [inline content/docs/deployment/csminstallationwizard/src/index.html:90]: @HarishH-DELL Please revert it completely.

**Review by rajendraindukuri — APPROVED** [review]

File: content/docs/deployment/csmoperator/release/_index.md [hunk 3]
```diff
@@ -31,4 +31,39 @@ Description: >
 - [#990 - [BUG]: X_CSI_AUTH_TYPE cannot be set in CSM Operator](https://github.com/dell/csm/issues/990)
 
 ### Known Issues
-There are no known issues in this release.
\ No newline at end of file
+
+| Issue | Workaround |
+|-------|------------|
+| It appears that the images of sideCars are currently missing in the sample YAMLs in the offline bundle. As a consequence, the csm-operator is pulling them from registry.k8s.io. | We recommend manually updating the images of sideCars in the sample YAML file, for example, `storage_csm_powerflex_v290.yaml`, before proceeding with the driver installation. Here is an example snippet for the sideCars section in the YAML file:
```
Comment (bharathsreekanth) [inline content/docs/deployment/csmoperator/release/_index.md:37]: Remove "It appears that"

**Review by bharathsreekanth — CHANGES_REQUESTED** [review]

**Review by adarsh-dell — DISMISSED** [review]

File: content/docs/deployment/csmoperator/release/_index.md [hunk 4]
```diff
@@ -31,4 +31,39 @@ Description: >
 - [#990 - [BUG]: X_CSI_AUTH_TYPE cannot be set in CSM Operator](https://github.com/dell/csm/issues/990)
 
 ### Known Issues
-There are no known issues in this release.
\ No newline at end of file
+
+| Issue | Workaround |
+|-------|------------|
+| The images of sideCars are currently missing in the sample YAMLs in the offline bundle. As a consequence, the csm-operator is pulling them from registry.k8s.io. | We recommend manually updating the images of sideCars in the sample YAML file, for example, `storage_csm_powerflex_v290.yaml`, before proceeding with the driver installation. Here is an example snippet for the sideCars section in the YAML file:
```
Comment (AkshaySainiDell) [inline content/docs/deployment/csmoperator/release/_index.md:37]: Let us change storage_csm_powerflex_v290.yaml to storage_csm_powerflex_v291.yaml

**Review by AkshaySainiDell — CHANGES_REQUESTED** [review]

File: content/docs/deployment/csmoperator/release/_index.md (hunk 4 above)
Comment (HarishH-DELL) [reply content/docs/deployment/csmoperator/release/_index.md]: fixed

**Review by AkshaySainiDell — APPROVED** [review]

**Review by adarsh-dell — APPROVED** [review]

**Review by cbartoszDell — APPROVED** [review]

**Review by mareksuski-dell — APPROVED** [review]
LGTM

**Pull request merged**